src/components/Card/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { Card, CardProps, CardRatioOptions } from './Card';
+export { OverlappingCard } from './OverlappingCard';
```

Comment:
we don't expose more than one component per folder.
please create a separate folder and docs for it